conf/compose/docker-compose.memcached.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.8"
 services:
   memcached:
-    image: memcached:latest
+    image: memcached:1.6.32
```

Comment:
Glad you added `stable` for nginx 👍 
For memcached, would it make sense to do the same thing like `1.6-bookworm`? That way it picks up bugfix-level releases